src/Uno.UI/UI/Xaml/Media/Brush.cs

```diff
@@ -14,24 +14,16 @@ public Brush()
 			InitializeBinder();
 		}
 
-		public static implicit operator Brush(Color uiColor)
-		{
-			Color color = uiColor;
-
-			return SolidColorBrushHelper.FromARGB(color.A, color.R, color.G, color.B);
-		}
+		public static implicit operator Brush(Color uiColor) => SolidColorBrushHelper.FromARGB(uiColor.A, uiColor.R, uiColor.G, uiColor.B);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Refactor this method to add validation of parameter 'uiColor' before using it.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5093276)